Makefile

```diff
@@ -61,7 +61,7 @@ changelog:
 
 # http://amazon-linux-2-packages.bref.sh/
 amazonlinux-package-list:
-	docker run --rm -it amazonlinux:2 yum list --quiet --color=never > index.html
+	docker run --rm -it --entrypoint= public.ecr.aws/lambda/provided:al2 yum list --quiet --color=never > index.html
```

Comment:
Should it really be an extra space here?

Reply:
https://github.com/brefphp/bref/pull/836

Reply:
I think so yes, this is to disable the `entrypoint`. The next argument is the Docker image, unless I'm mistaken.

Reply:
@Nyholm yes we need the extra space in... we can put `--entrypoint=scratch` for example like in extra extensions if it make things a little more readable/clear.